packages/core/src/components/sections/Navbar/section.module.scss

```diff
@@ -29,6 +29,15 @@
 
     @include media(">=notebook") {
       height: var(--fs-navbar-height-desktop);
+
+      // Sets margin to avoid overlapping the section below
+      &:has([data-fs-navbar-scroll]) {
```

Comment:
Using `has()` since it's being supported since dec 2023.
<img width="1238" alt="image" src="https://github.com/user-attachments/assets/0ca91fe2-033f-45a0-b3c4-9dfa83b03267" />